src/viur/core/bones/string.py

```diff
@@ -331,7 +331,13 @@ def refresh(self, skel: "SkeletonInstance", bone_name: str) -> None:
         # TODO: duplicate code, this is the same iteration logic as in NumericBone
         new_value = {}
         for _, lang, value in self.iter_bone_value(skel, bone_name):
-            new_value.setdefault(lang, []).append(self.type_coerce_single_value(value))
+            value = self.type_coerce_single_value(value)
+            if self.escape_html:
+                value = utils.string.escape(value)
+            else:
```

Comment:
But wouldn't this lead to 

"hello & bye" --> "hello &amp; bye" --> "hello &amp;amp; bye" --> "hello &amp;amp; bye"


```suggestion
            if not self.escape_html:
```

Reply:
`&` is not translated in `escape`